js/build-env-config.js

```diff
@@ -13,6 +13,7 @@ if (environment !== `PRODUCTION`) {
 }
 
 var config = {
+  ORIGIN: habitat.get(`ORIGIN`),
```

Comment:
As there is no `.env` file, this feels like it needs local dev defaults. `ORIGIN: habitat.get('ORIGIN', 'http://localhost:3000')`?